pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -63,6 +63,16 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 		return nil
 	}
 
+	// Refreshing cache info about target OS
+	isSUSE, err := target.IsSUSEOS()
+	if err != nil {
+		return err
+	}
+	// Check if target node OS is valid
+	if !isSUSE {
+		err = errors.New(fmt.Sprintf("Invalid target node OS: %s", nodeVersionInfoUpdate.Current.Node.Status.NodeInfo.OSImage))
```

Comment:
Suggest to use `fmt.Errorf` is much simpler.

Reply:
that can even be bundled in the return stanza as:
`return fmt.Errorf("Invalid target node OS: %s", nodeVersionInfoUpdate.Current.Node.Status.NodeInfo.OSImage)`

Reply:
Thanks a lot for hints, guys!